sickbeard/providers/rarbg.py

```diff
@@ -130,11 +130,16 @@ def search(self, search_strings, age=0, ep_obj=None):  # pylint: disable=too-man
                     continue
 
                 error = data.get("error")
-                error_code = data.get("error_code")
+                error_code = try_int(data.get("error_code"))
```

Comment:
@fernandog can you check if removing the try_int works? The other should then also be changed to if error_code not in ('20', '14'). but that should not be a problem.